lib/gyro/utils/file_utils.rb

```diff
@@ -26,4 +28,44 @@ def self.write_file_with_name(dir, name_file, content)
     file_path = File.expand_path(name_file, dir)
     File.write(file_path, content)
   end
+
+  def self.templates_dir
+    Pathname.new(File.dirname(__FILE__)) + '../../'
```

Comment:
Either:
* include the `/templates` part in that path if you name that method `templates_dir`, so that it actually indeed ponts to the templates directory
* or name that method `self.lib_dir` if you're not pointing to the actual template dir but the lib dir

(I personally prefer the first solution, keeping that name and pointing to `lib/templates`)